docs/source/kinds_of_types.rst

```diff
@@ -522,6 +521,21 @@ assigning the type to a variable:
     another type -- it's equivalent to the target type except for
     :ref:`generic aliases <generic-type-aliases>`.
 
+Since Mypy 0.930 you can also use explicit type aliases which are defined by :pep:`613`.
+
+Implicit type alias declaration rules create confusion when type aliases involve forward references,
+invalid types, or violate other restrictions enforced on type alias declaration.
+Because the distinction between an unannotated value and a type alias is implicit,
+ambiguous or incorrect type alias declarations implicitly default to a valid value assignment.
+
+.. code-block:: python
+
+   from typing import TypeAlias  # or `from typing_extensions` before `python3.10`
+
+   AliasType: TypeAlias = Union[list[dict[tuple[int, str], set[int]]], tuple[str, list[str]]]
+
+Explicit type aliases solves the ambiguness and improves readability.
```

Comment:
```suggestion
Explicit type aliases are unambiguous and improve readability.
```
"ambiguess" is not a word :)

Reply:
Thanks! Sometimes I make quite funny typos 🙂